Classes/Service/SolrServiceProvider.php

```diff
@@ -357,6 +364,10 @@ protected function addFacetQueries(): void
                     if (1 == $facet['excludeOwnFilter']) {
                         $queryForFacet->addExclude($this->tagForFacet($facetID));
                     }
+
+                    if ($facet['showmissing'] == 1) {
```

Comment:
Please use strict comparison. I know, the code part above does not use it, but we're about to change the strictness of the extension.